src/main/java/frc/team2412/robot/commands/climb/PostClimbDownComamnd.java

```diff
@@ -0,0 +1,24 @@
+package frc.team2412.robot.commands.climb;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.team2412.robot.subsystem.PostClimbSubsystem;
+
+public class PostClimbDownComamnd extends CommandBase {
+
+    private PostClimbSubsystem subsystem;
+
+    public PostClimbDownComamnd(PostClimbSubsystem subsystem) {
+        this.subsystem = subsystem;
+    }
```

Comment:
Should 'require' the subsystem so another command isn't scheduled at the same time that also uses it.